src/Kunstmaan/MediaBundle/Helper/ExtensionGuesserFactory.php

```diff
@@ -2,24 +2,25 @@
 
 namespace Kunstmaan\MediaBundle\Helper;
 
-use Kunstmaan\MediaBundle\Helper\File\SVGExtensionGuesser;
-use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesser;
-use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface;
+use Symfony\Component\Mime\MimeTypes;
+use Symfony\Component\Mime\MimeTypesInterface;
 
+@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6, use "%s" instead.', ExtensionGuesserFactory::class, MimeTypes::class), E_USER_DEPRECATED);
```

Comment:
```suggestion
@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6 and will be removed in KunstmaanMediaBundle 6.0. Use the symfony/mime component instead.', __CLA
SS__), E_USER_DEPRECATED);
```

Same for the deprecation message bellow